lib/pry/helpers/base_helpers.rb

```diff
@@ -60,9 +60,20 @@ def heading(text)
         "\e[1m#{text}\e[0m"
       end
 
-      # have fun on the Windows platform.
+      #
+      # @return [Boolean]
+      #   Returns true if Pry is running on Linux.
+      #
+      def linux?
+        !!(RbConfig::CONFIG['host_os'] =~ /linux/i)
```

Comment:
On Ubuntu this string is "linux-gnu". 
I don't know how consistent it is across different distributions. 
The regexp is lenient like the Windows one for that reason.

Reply:
cool but is this used anywhere?